webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr/NgffMetadata.scala

```diff
@@ -36,8 +36,11 @@ object NgffMetadata {
       mag =>
         NgffDataset(
           path = mag.toMagLiteral(allowScalar = true),
-          List(NgffCoordinateTransformation(
-            scale = Some(List[Double](1.0) ++ (dataSourceVoxelSize.factor * Vec3Double(mag)).toList)))
+          List(
+            NgffCoordinateTransformation(
+              scale = Some(List[Double](1.0) ++ (dataSourceVoxelSize.factor * Vec3Double(mag)).toList),
+              translation = Some(List[Double](0.0, 0.0, 0.0))
```

Comment:
Why add a zeroed translation by default? Is this in any way better than leaving out the translation?

Reply:
Actually None is better because the type by default is scale, so this would add translation to a scale which does not make a lot of sense